entity-framework/core/modeling/types-and-tables.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Types and Tables - EF Core
+author: roji
+ms.date: 12/03/2019
+ms.assetid: cbe6935e-2679-4b77-8914-a8d772240cf1
+uid: core/modeling/types-and-tables
+---
+# Types and Tables
+
+Including a type on your context means that it is included in EF Core's model; we usually refer to such a type as an *entity*. EF Core can read and write entity instances from/to the database, and if you're using a relational database, EF Core can create tables for your entities via migrations.
+
+## Including types in the model
+
+By convention, types that are exposed in DbSet properties on your context are included in the model as entities. Entity types that are specified in the `OnModelCreating` method are also included, as are any types that are found by recursively exploring the navigation properties of other discovered entity types.
+
+In the code sample below, all types are included:
+
+* `Blog` is included because it's exposed in a DbSet property on the context.
+* `Post` is included because it's discovered via the `Blog.Posts` navigation property.
+* `AuditEntry` because it is specified in `OnModelCreating`.
+
+[!code-csharp[Main](../../../samples/core/Modeling/Conventions/TypesAndTables.cs?highlight=3,7,16)]
+
+## Excluding types from the model
+
+If you don't want a type to be included in the model, you can exclude it:
+
+# [Data Annotations](#tab/data-annotations)
+
+[!code-csharp[Main](../../../samples/core/Modeling/DataAnnotations/IgnoreType.cs?highlight=20)]
+
+# [Fluent API](#tab/fluent-api)
```

Comment:
Suggest using `##` to avoid violating rule only 1 H1 heading.

Reply:
Somehow I didn't think of that... Will do it with three `###` so as to properly nest them in the inner title. Will also do it for the rest of the docs in a separate PR.